### ISSUE TITLE
Hyper-V modules need to be invoked with correct case

### ISSUE THREAD

**Issue by gbraad** [kubernetes/minikube#2639]
https://github.com/kubernetes/minikube/blob/a000c35e17dbf74c571f439c41171343fa5ab67f/vendor/github.com/docker/machine/drivers/hyperv/hyperv.go#L131 and others

this needed to be fixed with:
```diff
-	stdout, err := cmdOut("(", "hyper-v\\Get-VM", d.MachineName, ").state")
+	stdout, err := cmdOut("(", "Hyper-V\\Get-VM", d.MachineName, ").state")
```
else issues occured on the pre-create check

Full change can be seen in https://github.com/docker/machine/pull/4426

**Comment by gbraad** [comment]
It is hard to track which version you actually have merged, but it seems it is somewhere after 0.14.0 ?

AFAIR this is only used in Docker for Windows/Mac edge releases.

**Comment by saitho** [comment]
The bugfix in docker-machine is currently only in the master branch and has not been released in a version yet... I'm currently getting the error "Hyper-V PowerShell Module is not available" when I'm trying to start Minikube (for the first time) on my Windows computer with Hyper-V enabled.

I compiled docker-machine from the master branch, but still can't start up Minikube. ^^ (The issue described on the docker-machine project works fine now).

**Comment by saitho** [comment]
I solved the issue locally and recompiled minikube. Seems to work fine now.
However I did not decide to open a PR for that as I'm pretty new to Go and I have no idea about how dependency management exactly works... For now I just adjusted the files in the vendor/ directory. ^^

I can confirm that the problem described in #2634 is solved with that. I was able to start Minikube successfully on my Windows computer with Hyper-V enabled.
